etc/Makefile.am

```diff
@@ -60,6 +60,6 @@ EXTRA_DIST = \
   rm_bgq_slurm.conf \
   rm_mchecker.conf \
   rm_openrte.conf \
-  rm_slurm.conf \
+  rm_slurm.conf.in \
```

Comment:
A bit worried since this is one off. Do you mind adding a one line comment somewhere as to why <conf.in> was introduced? Or that explanation can go to a commit message if that's simpler.

Reply:
Good idea, I added to the update log of the modified files.